### ISSUE TITLE
Location services does not start correctly

### ISSUE THREAD

**Issue by itamaker** [malcommac/SwiftLocation#61]
First into the application without authorization, or executing a method
info.plist config：NSLocationWhenInUseUsageDescription

``` swift
Location.getLocation(withAccuracy: .Block, onSuccess: { foundLocation in

})  { (lastValidLocation, error) in
           // something bad has occurred
           // - error contains the error occurred
          // - lastValidLocation is the last found location (if any) regardless specified accuracy level
 }
```

**Comment by malcommac** [comment]
Hi, @itamaker, what's the problem?

**Comment by malcommac** [comment]
Location services does not start correctly and no events is reported.
Fixed in 1.0.3 currently pushed both on CocoaPods and Carthage.
